sled-agent/src/bootstrap/agent.rs

```diff
@@ -434,6 +540,66 @@ impl Agent {
         &self.baseboard
     }
 
+    async fn poll_ddmd_for_bootstore_peer_update(
+        log: &Logger,
+        bootstore_node_handle: bootstore::NodeHandle,
+    ) -> JoinHandle<()> {
+        let log = log.clone();
+        tokio::spawn(async move {
+            // Explicit fail fast. We can't do anything without a DdmAdminClient.
+            // Do we need to loop as in wicketd?
+            let ddmd_client = DdmAdminClient::localhost(&log).unwrap();
```

Comment:
This feels like a TODO? Is it possible for this to fail?

Reply:
Yeah, I would assume it's possible to fail. I should definitely just add a retry to handle the case of spurious failures.

Reply:
Retries implemented in [d8fcf7e](https://github.com/oxidecomputer/omicron/commit/d8fcf7eecf302be91d647f228708259e2123fc23)